test/functional/common/phases.js

```diff
@@ -22,13 +22,13 @@ function confirmation (goodTransactions, badTransactions, pendingMultisignatures
 					'id=' + transaction.id
 				];
 				return apiHelpers.getTransactionsPromise(params).then(function (res) {
-					expect(res).to.have.property('status').to.equal(200);
-					expect(res).to.have.nested.property('body.transactions').to.be.an('array').to.have.lengthOf(0);
+					res.body.data.should.have.length(0);
 				});
 			});
 		});
 
-		it('good transactions should not be unconfirmed', function () {
+		// TODO: After migration /transactions/unconfirmed make sure this phase works
+		it.skip('good transactions should not be unconfirmed', function () {
```

Comment:
@karmacoma Have to skip, as unconfirmed transactions endpoint is removed temporarily.

Reply:
Ok